src/MaintenanceModeCommand.php

```diff
@@ -2,6 +2,154 @@
 
 namespace WP_CLI\MaintenanceMode;
 
-class MaintenanceModeCommand extends WP_CLI_Command {
+/**
+ * Activates, deactivates or checks the status of the maintenance mode of a site.
+ *
+ * ## EXAMPLES
+ *
+ *     # Activate Maintenance mode.
+ *     $ wp maintenance-mode activate
+ *     Enabling Maintenance mode...
+ *     Success: Activated Maintenance mode.
+ *
+ *     # Deactivate Maintenance mode.
+ *     $ wp maintenance-mode deactivate
+ *     Disabling Maintenance mode...
+ *     Success: Deactivated Maintenance mode.
+ *
+ *     # Display Maintenance mode status.
+ *     $ wp maintenance-mode status
+ *     Maintenance mode is active.
+ *
+ *     # Get Maintenance mode status for scripting purpose.
+ *     $ wp maintenance-mode is-active
+ *     $ echo $?
+ *     1
+ *
+ * @when after_wp_load
+ * @package wp-cli
+ */
+class MaintenanceModeCommand extends \WP_CLI_Command {
 
+
+	/**
+	 * Instance of the class.
+	 *
+	 * @var object Instance of the class.
+	 */
+	private static $instance;
+
+	/**
+	 * Return instance of the class.
+	 *
+	 * @return object
+	 */
+	public static function get_instance() {
+		if ( ! isset( self::$instance ) ) {
+			if ( ! class_exists( '\WP_Upgrader' ) ) {
+				require_once ABSPATH . 'wp-admin/includes/class-wp-upgrader.php';
+			}
+			self::$instance = new \WP_Upgrader( new \WP_CLI\UpgraderSkin() );
```

Comment:
You should use imports at the top, so one can see at a glance what the dependencies are for this class.
```suggestion
			self::$instance = new WP_Upgrader( new WP_CLI\UpgraderSkin() );
```